src/include/network/network_io_utils.h

```diff
@@ -221,6 +221,13 @@ class ReadBufferView {
     return result;
   }
 
+  /**
+   * @param bytes The amount of bytes between the current cursor and rest of the buffer view
+   * @return Whether there is any more bytes between the cursor and
+   *         the end of the buffer view
+   */
+  bool HasMore(size_t bytes = 1) { return offset_ + bytes <= size_; }
```

Comment:
This is identical to Buffer::HasMore, but I understand that you might need similar functionality in the ReadBufferView.

That said, we're trying to move away from having default arguments. Consider rewriting to `HasMore(size_t bytes)` and maybe if you really hate calling `HasMore(1)` then you can have an `IsFinished` or something like that which calls `HasMore(1)` for you.